src/styles.scss

```diff
@@ -10,8 +14,8 @@ html {
 body {
   margin: 0;
   font-family: "Open Sans", sans-serif;
-  background-color: #F7F7F7;
-  color: #333E48;
+  background-color: #f7f7f7;
```

Comment:
Can you please use the predefined color variables across the application? (Not just here, but anywhere else)
Color variables are under /shared/styles/variables/colors.scss You can add your own color, too.